### ISSUE TITLE
tick size doesn't scale

### ISSUE THREAD

**Issue by ElKatz** [jcbyts/pdstools#6]
the x & y ticks on the abscissa & ordinate do not scale with the xy lims of the plot resulting in huge when setting the xy lims to a small value, e.g. xlim([0 0.05])

![screen shot 2015-10-11 at 5 56 23 pm](https://cloud.githubusercontent.com/assets/5068044/10419580/697c1396-7041-11e5-8877-6afdd31f8f70.png)

**Comment by jcbyts** [comment]
Right. That's because they are percents of the scatter plot range... hmm. I believe they should scale for values above 1. Is it totally busted or just busted when the numbers get small enough?

**Comment by jcbyts** [comment]
Yea, tick size is totally messed up. I don't know an easy way to fix this right now. Any ideas?

**Comment by ElKatz** [comment]
right now the scaling is off. below are three orders of std mag.
I think we can add xlim/ylim as input args and do it through that

![screen shot 2015-10-11 at 6 11 33 pm](https://cloud.githubusercontent.com/assets/5068044/10419659/8dff416e-7043-11e5-96ad-e99607b7deb6.png)
